Lucidtech/Las/Client.cs

```diff
@@ -1268,6 +1295,10 @@ private static bool FatalCode(HttpStatusCode code)
         
         private object JsonDecode(IRestResponse response)
         {
+            if (response.StatusCode == HttpStatusCode.NoContent)
+            {
+                return null;
```

Comment:
In Java SDK we are returning a client constructed response, should we not do the same here?